source/_integrations/overkiz.markdown

```diff
@@ -7,6 +7,7 @@ ha_category:
   - Button
   - Lock
   - Number
+  - Scene
 ha_release: 2022.1.0
```

Comment:
```suggestion
ha_release: 2022.2.0
```

Reply:
For a seperate PR

Reply:
https://github.com/home-assistant/home-assistant.io/pull/20881